components/centraldashboard/app/k8s_service.ts

```diff
@@ -7,39 +7,31 @@ export interface PlatformInfo {
   kubeflowVersion: string;
 }
 
-/**
- * Relevant fields from the description property of the Application CRD
- * https://github.com/kubernetes-sigs/application/blob/master/config/crds/app_v1beta1_application.yaml
- */
-interface V1BetaApplicationDescriptor {
-  version: string;
-  type: string;
-  description: string;
+interface Operator {
+  kind: string;
+  name: string;
 }
 
-/**
- * Relevant fields from the spec property of the Application CRD
- * https://github.com/kubernetes-sigs/application/blob/master/config/crds/app_v1beta1_application.yaml
- */
-interface V1BetaApplicationSpec {
-  descriptor: V1BetaApplicationDescriptor;
+interface OperatorVersionSpec {
+  appVersion: string;
+  operator: Operator;
 }
 
-/** Generic definition of the Kubeflow Application CRD */
-interface V1BetaApplication {
+/** Generic definition of the OperatorVersion CRD */
+interface OperatorVersion {
   apiVersion: string;
   kind: string;
   metadata?: k8s.V1ObjectMeta;
-  spec: V1BetaApplicationSpec;
+  spec: OperatorVersionSpec;
 }
 
-interface V1BetaApplicationList {
-  items: V1BetaApplication[];
+interface OperatorVersionList {
```

Comment:
shall we prepend the interface names with API version as well?

Reply:
done